src/app/owner/page.tsx

```diff
@@ -0,0 +1,90 @@
+import SubHeading from '../components/OwnerPage/SubHeading';
+import Section from '../components/OwnerPage/Section';
+import workMethodsContent from '../utils/workMethodsContent';
+import { educationAndExperience, certificatesAndAchievements } from '../utils/aboutContent';
+import Container from '../components/Share/Container';
+import Button from '../components/Share/Button';
+
+const About = () => {
+  return (
+    <div className="min-h-screen bg-white">
+      <Container>
+        <main>
+          <nav className="text-[16px] mb-4 mt-[70px]">
+            <a href="/" className="font-semibold leading-6 underline hidden md:block" aria-label="Wróć do strony głównej">powrót</a>
+          </nav>
+
+          <p className="font-bold text-[16px] text-sm leading-6 mt-[80px] md:mt-[29px]">O nas // Katarzyna Bober</p>
+          <h1 className="text-center font-dm-sans font-medium text-[44px] leading-[52.8px] tracking-[-0.04em] mt-[40px]">Katarzyna Bober</h1>
+
+          <div className="flex justify-center mb-[20px]">
+            <Image
+              src={about as StaticImageData}
+              alt="Katarzyna Bober with dog"
+              width={1217}
+              height={600}
+              className="rounded-lg mt-[100px] max-w-full"
+              priority
+            />
+          </div>
+
+          <div className="font-normal text-sm leading-6 text-left mb-[80px] mt-[100px]">
+            <AboutHeading>O mnie</AboutHeading>
+
+            <SubHeading>Wykształcenie i doświadczenie zawodowe</SubHeading>
+            <ul className="list-disc ml-6">
+              {educationAndExperience.map((item, index) => (
+                <li key={index}>
+                  {item.title}: {item.description}
+                </li>
+              ))}
+            </ul>
+            <SubHeading>Certyfikaty i osiągnięcia</SubHeading>
+            <ul className="list-disc ml-6">
+              {certificatesAndAchievements.map((item, index) => (
+                <li key={index}>
+                  {item.title}: {item.description}
+                </li>
+              ))}
+            </ul>
+            <AboutHeading>Jak pracuję</AboutHeading>
+            {workMethodsContent.map((item, index) => (
+              <Section
+                key={index}
+                title={item.title}
+                content={item.content}
+              />
+            ))}
+
+            <p className='mt-[40px]'>W Salonie Pielęgnacji Zwierząt Spoon Dog stosuję, wyłącznie oryginalne kosmetyki, akcesoria i narzędzia, stworzone specjalnie do pracy ze zwierzętami <span className='font-medium'>firm Oster, Andis i Wahl.</span> Służę  <span className='font-medium'>poradami behawioralnymi, żywieniowymi i pielęgnacyjnymi,</span> a o zwierzętach potrafię rozmawiać godzinami. </p>
+
+            <p className='font-medium mt-[40px]'> Kilkuletnie doświadczenie sprawia, że możecie Państwo mi w pełni zaufać. </p>
+
+            <p className='hidden lg:block mt-[40px]'>Zapraszam</p>
```

Comment:
“lg:” breakpoint is not applicable in this project, as it applies to resolutions of 1024px upwards. To properly assign styles for the desktop, use “desktop:”, which applies to resolutions of 1440px and up.